package.json

```diff
@@ -69,7 +69,7 @@
     "@types/tryghost__content-api": "^1.3.10",
     "cli-progress": "^3.11.0",
     "eslint": "^8.11.0",
-    "firebase-tools": "^10.8.0",
+    "firebase-tools": "^11.0.1",
```

Comment:
This requires Java 11 to be installed so I want to test it first and then update the READMEs.